src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.targets

```diff
@@ -20,6 +20,12 @@
     <CLSCompliant Condition="'$(CLSCompliant)'=='' and '$(IsTestProject)'=='true'">false</CLSCompliant>
     <CLSCompliant Condition="'$(CLSCompliant)'==''">true</CLSCompliant>
 
+    <!--
+      Find projects in the tests folder which are not executable to distinguish
+      from accompanying support libraries.
+    -->
+    <IsRealTestProject Condition="'$(IsTestProject)' == 'true' AND '$(OutputType.ToLower())' != 'exe'">true</IsRealTestProject>
```

Comment:
I'd use a different word than "real" here.  `IsExecutableTestProject` maybe?

Reply:
It's exactly the opposite. It's not executable by default. `IsRunnableTestProject` or `IsLibTestProject`?

Reply:
Ok I'll go with `IsInvokableTestProject`?

Reply:
Good enough for me.